src/kata6.joinNames.js

```diff
@@ -1,3 +1,13 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  const namesArray = namesObj.map((person) => person.name);
+
+  const stringOfNames = namesArray.reduce((acc, curr, index) => {
+    if (index === namesArray.length - 1) {
+      return acc + " & " + curr;
+    }
+    return acc + ", " + curr;
+  });
+  return stringOfNames;
+};
```

Comment:
Nice approach! Theres a couple of things to improve here, you dont need to create the variable on line 4 only to return it on line 10, you can just return the value of the function directly. Also, eslint is not happy about the formatting of lines 6 and 8, as template literal notation is preferred here. PUttin ghtese together you would end up with this:
```
const joinNames = (namesObj) => {
  const namesArray = namesObj.map((person) => person.name);

  return namesArray.reduce((acc, curr, index) => {
    if (index === namesArray.length - 1) {
      return `${acc} & ${curr}`;
    }
    return `${acc}, ${curr}`;
  });
};
```